server.js

```diff
@@ -27,9 +29,69 @@ const app = express()
 app.use(cors())
 app.use(bodyParser.json())
 
+
+const Data = new mongoose.model('Data', {
+    year_film: Number,
+    year_award: Number,
+    ceremony: Number,
+    category: String,
+    nominee: String,
+    film: String,
+    win: Boolean
+})
+
+//async await handles the asynchronous code (because we are using a database). By doing this we first clear the database 
+// and it doesn't proceed with code on line 46 until the await on code 44 is finished. This is how we avoid adding the same data to the database all the time.
+if (process.env.RESET_DATABASE) {
+	const seedDatabase = async () => {
+    await Data.deleteMany()
+
+	goldenGlobesData.forEach((nominationData) => {
+    const newData = new Data(nominationData)
+      newData.save()
+	})
+}
+  seedDatabase()
+}
+
 // Start defining your routes here
 app.get('/', (req, res) => {
-  res.send('Hello world')
+  res.send('Golden Globes nominations API in MongoDB')
+})
+
+//Shows all nominations. We can also query for information, for example to see all the movies that won we have to add to the URL: ?win=true. We can also add other things like for example the name of the film: ?win=true&film=Avatar
+app.get('/nominations', async (req, res) => {
+  const queryParameters = req.query
+  const allNominations = await Data.find(req.query)
+  res.json(allNominations)
+})
+
+//This will return one object with a specified actor, for example 'Sandra Bullock', if there is no such name it will return an error message. If we use Find instead of FindOne it will return all
+// the objects with Sandra Bulllock as nominee. Works also with lowercase: sandra bullock.
+app.get('/nominations/nominee/:nominee', async (req, res) => {
+  const nominee = req.params.nominee
+  const singleNominee = await Data.findOne({nominee: { $regex: nominee, $options: 'i'}})
+
+  if (!singleNominee) {
+    res.status(404).json(ERROR)
+  } else {
+    res.json(singleNominee)
+  }
+})
```

Comment:
`nominee` is not a good key for extracting single document, unfortunately. When you use it like this you returning a bad set of data because you cutting off a big portion of it. For example "Avatar" has a lot of nominations, but you are showing only the first one.

You need to use field which is always unique for every document, usually, it is `id` field.

Reply:
Since there aren't any id:s for this dataset I deleted the endpoint and added another one on line 70: /nominations/:year/:category/winner.

Reply:
Yes, you have it. Mongoose creates it automatically and names it `_id` you can see it here: https://project-mongo-api-ingela.herokuapp.com/nominations 
The easiest way to connect to it would be to use https://mongoosejs.com/docs/api.html#model_Model.findById

Reply:
Thanks! Think I got it now. Just pushed new version with endpoint /nominations/nominee/:id